### PR TITLE
[zh] sync /contributor/lifted.md

### PR DESCRIPTION
**Pull request by windsonsea**
Added zh text to:

```
 i18n/zh/docusaurus-plugin-content-docs/current/contributor/lifted.md
```

/kind documentation

/cc @Poor12

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/contributor/lifted.md [hunk 1]
```diff
@@ -1,39 +1,43 @@
 ---
-title: How to manage lifted codes
+title: 如何管理 Lift 代码
```
Comment (Poor12) [inline i18n/zh/docusaurus-plugin-content-docs/current/contributor/lifted.md:2]: Lift or Lifted？
What do you think is better?

Comment (windsonsea) [reply i18n/zh/docusaurus-plugin-content-docs/current/contributor/lifted.md:2]: hmmm, this page has two forms of lift:
- lifted
- lifting
Less tenses are found in Chinese. In fact, I don't know how to translate `lift`, so just leave it as is. 😅

Comment (Poor12) [reply i18n/zh/docusaurus-plugin-content-docs/current/contributor/lifted.md:2]: Okay.

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**